src/main/java/bio/terra/service/profile/ProfileService.java

```diff
@@ -130,7 +131,7 @@ public EnumerateBillingProfileModel enumerateProfiles(Integer offset,
                                                           AuthenticatedUserRequest user) {
         List<UUID> resources = iamService.listAuthorizedResources(user, IamResourceType.SPEND_PROFILE);
         if (resources.isEmpty()) {
-            return new EnumerateBillingProfileModel().total(0);
+            return new EnumerateBillingProfileModel().items(Collections.emptyList()).total(0);
```

Comment:
```suggestion
            return new EnumerateBillingProfileModel().total(0).items(Collections.emptyList());
```

Should it be this to be consistent with your other usage?

Reply:
oh sure, fair enough